### DISCUSSION TITLE
Too many Flockbits!

### DISCUSSION THREAD

**Discussion by stonepillars** [stonepillars/goon-flock#307] [Ideas]
It's very easy to make **alot of them**.
and I mean, ***alot of them***
![image](https://user-images.githubusercontent.com/65367576/169699570-2e350f79-532c-4155-b7de-ce58f49b8881.png)

Should there be some sort of unit cap? Cooldown on the diffraction ability?
Having lots of bits also contributes to lag from flock running

**Comment by amylizzle** [comment]
10 second cooldown would probably solve this, but also we can make the building behaviour a little more efficient
